.travis.yml

```diff
@@ -1,13 +1,11 @@
-language: cpp
+sudo: required
+language: ruby
+services:
+  - docker
+
 before_install:
-    # disable rvm, use system Ruby
-    - rvm reset
-    - wget https://raw.githubusercontent.com/yast/yast-devtools/master/travis-tools/travis_setup.sh
-    - sh ./travis_setup.sh -p "ruby2.1 rake yast2-devtools" -g "yast-rake yard gettext"
+  - docker build -t yast-test-image .
```

Comment:
@lslezak I have tried to run this locally by first building the https://github.com/yast/docker-yast-ruby-tw image locally. The problem is that this uses the same name `yast-test-image` both for the common image and the image with the git repo being tested in /tmp/sources. I think the downstream image should have a different name.